template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProviderTest.java

```diff
@@ -34,19 +40,54 @@
     private final RangerCloudStorageServiceConfigProvider underTest = new RangerCloudStorageServiceConfigProvider();
 
     @Test
-    public void testGetRangerCloudStorageServiceConfigs() {
+    public void testGetRangerAWSCloudStorageServiceConfigs() {
         CmTemplateProcessor templateProcessor = mock(CmTemplateProcessor.class);
-        TemplatePreparationObject preparationObject = getTemplatePreparationObject(true)
+        TemplatePreparationObject preparationObject = getTemplatePreparationObjectForAWS(true)
                 .withBlueprintView(mock(BlueprintView.class))
+                .withProductDetails(new ClouderaManagerRepo().withVersion("7.2.1"), List.of())
                 .build();
 
         List<ApiClusterTemplateConfig> serviceConfigs = underTest.getServiceConfigs(templateProcessor, preparationObject);
 
         assertEquals(1, serviceConfigs.size());
-        assertEquals("ranger_plugin_hdfs_audit_url", serviceConfigs.get(0).getName());
+        assertEquals(RANGER_HDFS_AUDIT_URL, serviceConfigs.get(0).getName());
```

Comment:
This is not recommended. Your tests should not be importing these constants. The reason is that if you do this, then if someone makes a typo in "ranger_plugin_hdfs_audit_url", you won't be catching it. 

By making this constant explicit in the tests, a developer has to make a conscious decision on making the change for constants.

Reply:
That's probably not needed, these fields are not configurable.

Reply:
As a general principle, your tests shouldn't use imported constants. Yes they are not configurable but a dev can accidentally cause a regression by accidentally changing the constant.

It has happened before :)